core/src/test/scala/com/spingo/op_rabbit/ConnectionParamsMultihostSpec.scala

```diff
@@ -1,12 +1,15 @@
 package com.spingo.op_rabbit
 
 import com.rabbitmq.client.Address
-import com.typesafe.config.{Config, ConfigFactory, ConfigValue, ConfigValueFactory}
+import com.typesafe.config.{Config, ConfigFactory, ConfigValueFactory}
 import org.scalatest.{FunSpec, Matchers}
 
 import scala.collection.JavaConverters._
 
-class ConnectionParamsSpec extends FunSpec with Matchers {
+/**
+  * Created by Anton Naumov on 05.10.17.
```

Comment:
Where did this comment come from?

Reply:
That's some IntelliJ IDEA automatic stuff, thank you for pointing this I'll clean it up